examples/demo_vdc.pp

```diff
@@ -0,0 +1,22 @@
+# Manifest to demo cisco_vdc
+#
+# Copyright (c) 2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Once https://github.com/cisco/cisco-network-puppet-module/pull/129/files is merged you should add a platform check and only apply on supported platforms.  Also you can use this as a guide to create a new function to query the line card type.